### PR TITLE
update README to include note about wreck being at the top level

### PR DESCRIPTION
**Pull request by lloydbenson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ _'Monitor'_ should be configured using a _'hapi'_ server instead of calling the
 - `log` - logging information not bound to a specific request such as system errors, background processing, configuration errors, etc. Maps to the "log" event emitted from hapi servers.
 - `error` - request responses that have a status code of 500. This maps to the "request-error" hapi event.
 - `request` - Request logging information. This maps to the hapi 'request' event that is emitted via `request.log()`.
-- `wreck` - Wreck module request/response logging information.  
+- `wreck` - Wreck module request/response logging information.  Note: Wreck has to be installed at the top level in order for this to work due to it being a singleton.
```
Comment (AdriVanHoudt) [inline README.md]: What do you mean with top level? globally or just my project's node_modules?

Comment (lloydbenson) [inline README.md]: It does not need to be installed globally via -g.  When I saw top level, you may use wreck in many plugins in your overall project.  In order to see them, you need to install it at the top level project.  What wording would you prefer?

Comment (AdriVanHoudt) [inline README.md]: hmm can't find anything better :D

Comment (lloydbenson) [inline README.md]: How about that?

Comment (AdriVanHoudt) [inline README.md]: :+1: genious

**Pull request merged**